client/src/AppBundle/Resources/views/Macros/macros.html.twig

```diff
@@ -327,3 +341,72 @@
         </svg>
     </a>
 {% endmacro %}
+
+{% macro successBanner(successText, transDomain, successHeading) %}
+    <div class="moj-banner moj-banner--success">
+        <div class="moj-banner__message">
+            {% if successHeading is not null %}
+                <h3 class="govuk-heading-m">
```

Comment:
I think this should be `<h2>`, to make the heading structure:

- h1. Report overview
  - h2. More information needed
  - h2. Contacts
  - h2. Health and Welfare
  - ...

Reply:
Agree - I'll update.

Reply:
Has this been actioned? Looks like it's still an `<h3>`

Reply:
Think thats Github being out of date on the changes:

![Screenshot 2020-04-09 at 12 27 43](https://user-images.githubusercontent.com/47983793/78890420-8a5b2e00-7a5d-11ea-9904-ef1527aea1fc.png)